src/vm/types/layout.zig

```diff
@@ -153,6 +166,90 @@ pub const CairoLayout = struct {
         };
     }
 
+    /// Sets up the built-in runners for the layout.
+    ///
+    /// # Arguments
+    ///
+    /// - `self`: The layout structure.
+    /// - `allocator`: Allocator for memory management.
+    /// - `proof_mode`: Whether the program is in proof mode.
+    /// - `program_builtins`: The builtins used by the executed program.
+    ///
+    /// # Returns
+    ///
+    /// An array list of builtin runners.
+    pub fn setUpBuiltinRunners(
+        self: Self,
+        allocator: Allocator,
+        proof_mode: bool,
+        program_builtins: []const []const u8,
+    ) !ArrayList(BuiltinRunner) {
+        var builtin_runners = ArrayList(BuiltinRunner).init(allocator);
+
+        if (proof_mode) {
+            // TODO: initialize all the builtins in the layout.
+        }
+
+        const Case = enum {
```

Comment:
We already have a `BuiltinName` enum in `program.zig` file :)

Reply:
oops :D